### PR TITLE
copy update

### PR DESCRIPTION
**Pull request by jimaek**
Some copy changes

### CODE REVIEW

File: src/views/components/landing-top.html [hunk 1]
```diff
@@ -53,6 +53,7 @@ <h2>free, fast, and reliable</h2>
 							<c-slider active="{{activeSlide}}" direction="vertical" distance="500">
 								<c-slide>
 									<div class="docs-code-slide">
+										<p class="docs-comment">// NPM CDN</p>
```
Comment (MartinKolarik) [inline src/views/components/landing-top.html:56]: These look ugly and just duplicate the text directly below.

Comment (jimaek) [reply src/views/components/landing-top.html:56]: Its for SEO. Its the best and most relevant place I could think to add keywords

File: src/views/r-page.html [hunk 2]
```diff
@@ -10,11 +10,11 @@
 		<head>
 			<meta charset="utf-8">
 			<meta name="viewport" content="width=device-width, initial-scale=1">
-			<meta name="description" content="{{ description || 'A free, fast, and reliable Open Source CDN for npm and GitHub with the largest network and best performance among all CDNs. Serving more than 40 billion requests per month.'}}">
-			<meta name="keywords" content="jsdelivr, cdn, npm, github, js, javascript, free, open source, oss">
+			<meta name="description" content="{{ description || 'A free, fast, and reliable CDN for open source projects, deno, npm and GitHub with the largest network and best performance among all CDNs. Serving more than 80 billion requests per month.'}}">
```
Comment (MartinKolarik) [inline src/views/r-page.html:13]: "for open source projects, deno, npm" etc doesn't make much sense, if you don't want to call it open source cdn, we can just drop that part and let it be "reliable CDN for deno, npm, ..."

Comment (jimaek) [reply src/views/r-page.html:13]: But we are not limited only for them. We support any open source project with our custom configs.

Comment (MartinKolarik) [reply src/views/r-page.html:13]: Maybe but that's not the main focus, makes the text too complicated, and in no way explains what it means.

Comment (jimaek) [reply src/views/r-page.html:13]: how so? i think "a free cdn for open source projects" is pretty clear

File: src/views/components/landing-top.html [hunk 3]
```diff
@@ -10,8 +10,8 @@
 	<div class="content">
 		<div class="container">
 			<section class="heading">
-				<h1>jsDelivr – Open Source CDN</h1>
-				<h2>free, fast, and reliable</h2>
+				<h1>A free CDN for Open Source Projects</h1>
+				<h2>Fast, global, reliable and automated</h2>
```
Comment (MartinKolarik) [inline src/views/components/landing-top.html:14]: I also don't like this, both are too long. Maybe something like this

image

although I still like the current one better.

Comment (jimaek) [reply src/views/components/landing-top.html:14]: Ok lets use your shorter version

File: src/views/r-page.html (hunk 2 above)
Comment (MartinKolarik) [reply src/views/r-page.html:13]: Yes but when you mix it with the other terms it gets long and "open source projects" doesn't really fit in there "CDN for open source projects, deno, npm, and GitHub"

Comment (jimaek) [reply src/views/r-page.html:13]: Maybe its not ideal but I dont see any other way. Our site is not very clear or Google friendly. 
This way the user will just look at "CDN for" and then the keyword relevant to him and understand that it fits his need.

Comment (MartinKolarik) [reply src/views/r-page.html:13]: We have open source all over the place, including the main headline, it doesn't need to be everywhere.

Comment (jimaek) [reply src/views/r-page.html:13]: which always refers to us, not projects we host. Also this description is for Google where other mentions of oss are not visible

File: src/views/components/landing-top.html (hunk 1 above)
Comment (MartinKolarik) [reply src/views/components/landing-top.html]: Rephrased and moved to a better place.

**Pull request merged**